### ISSUE TITLE
Int overflow generates misleading error message

### ISSUE THREAD

**Issue by AchimStremplat** [COVESA/ramses-logic#33]
The script

```
function interface()
    IN.in_int = INT
    OUT.out_int = INT
    OUT.out_intplusone = INT
end

function run()
    OUT.out_int = IN.in_int
    OUT.out_intplusone = IN.in_int + 1
end
```

generates with the input in_int = 2147483647 the error 
`Implicit rounding during assignment of integer output 'out_intplusone' (value: 2147483600)!`
which is somewhat misleading - the 2147483600 is caused by converting the incoming number to float.

Not sure how to fix that - if it is possible to check for the type of the number maybe change the error message for an int?

**Comment by violinyanev** [comment]
Hi @AchimStremplat ,
sorry for the long delay... Yes, issue is clear, will make a better error output in the next release.

**Comment by violinyanev** [comment]
@AchimStremplat issue is fixed in v0.13.0. Merry Christmas! ;)
